src/metatrain/utils/omegaconf.py

```diff
@@ -316,7 +316,7 @@ def expand_dataset_config(conf: Union[str, DictConfig, ListConfig]) -> ListConfi
 
                 # for special case "energy" we enable sections for `forces` and `stress`
                 # gradients by default
-                if target_key == "energy":
+                if target_key == "energy" or target.get("quantity") == "energy":
```

Comment:
Should be enough no?

```suggestion
                if target.get("quantity") == "energy":
```

Reply:
No, "quantity" can be None at this point